recipes/packit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.24" %}
+{% set hash_value = "f22cbf399a5fcded061c566691be58d421b6c2b668ee0602665749a0266715f7" %}
+
+package:
+  name: python-packit
+  version: '{{ version }}'
+
+source:
+  url: https://files.pythonhosted.org/packages/58/4f/0ac86eb3cc9ce74f87de314faf33426798052b5b17a8e18bb4f0a08686b4/packit-0.24.tar.gz
+  sha256: '{{ hash_value }}'
+  patches:
+    - version.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel >=0.24
+    - pbr >=4.0
+    - glob2 >=0.4.1
+    - packaging >=16.8
+
+  run:
+    - python
+    - setuptools
```

Comment:
Does this project rely on ``setuptool`` to run or it is necessary just during installation?